deeppavlov/dataset_iterators/basic_classification_iterator.py

```diff
@@ -36,8 +36,9 @@ class BasicClassificationDatasetIterator(DataLearningIterator):
         field_to_split: name of field (out of ``"train", "valid", "test"``) to split
         split_fields: list of fields (out of ``"train", "valid", "test"``) to which save splitted field
         split_proportions: list of corresponding proportions for splitting
-        seed: random seed
+        seed: random seed for iterating
         shuffle: whether to shuffle examples in batches
+        split_seed: random seed for splitting dataset
```

Comment:
I think, it needs to be mentioned that if `split_seed` is `None`, it will be based on the `seed` parameter